From 24033d5d486074f17df237e098508852102d52b8 Mon Sep 17 00:00:00 2001
From: Simon Que <sque@chromium.org>
Date: Fri, 20 Apr 2012 16:23:59 -0700
Subject: [PATCH] chgrp /dev/input devices to chronos for DRM

Since chrome runs as chronos, this patch will let chrome access
/dev/input devices.

Change-Id: Ia6f61edcf9aaad8cf20a99e0a73c21add764a155
Signed-off-by: Simon Que <sque@chromium.org>
---
 mouse_added    |    1 +
 touchpad_added |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/mouse_added b/mouse_added
index be901f3..7b94797 100755
--- a/mouse_added
+++ b/mouse_added
@@ -6,4 +6,5 @@
 # Log all calls, with arguments, to /var/log/messages
 logger "$0" "$@" "$DEVNAME"
 
+chgrp chronos /dev/input/event* /dev/input/m*
 /opt/google/mouse/mousecontrol add &
diff --git a/touchpad_added b/touchpad_added
index f2fc2a7..10a7714 100755
--- a/touchpad_added
+++ b/touchpad_added
@@ -6,4 +6,5 @@
 # Log all calls, with arguments, to /var/log/messages
 logger "$0" "$@" "$DEVNAME"
 
+chgrp chronos /dev/input/event* /dev/input/m*
 /opt/google/touchpad/tpcontrol add &
-- 
1.7.3.4
